src/theme/style.scss

```diff
@@ -1,29 +1,81 @@
 @import url("https://fonts.googleapis.com/css2?family=Montserrat:wght@400;500;700&display=swap");
+$mycolor: "#83CE27";
 
 body {
-  font-family: "Montserrat", "sans-serif";
+  font-family: "serif", "sans-serif";
```

Comment:
I used `Montserrat` because it approximates Proxima Nova. This font set says "use serif font" (like Times New Roman), if there isn't a "serif" font use the default "sans-serif". There will never *not* be a serif font.